src/frontend/Komponenter/Behandling/Brev/Htmlfelter.tsx

```diff
@@ -1,17 +1,10 @@
 import { formaterNullableIsoDato, formaterTallMedTusenSkille } from '../../../App/utils/formatter';
 import { AndelTilkjentYtelse, TilkjentYtelse } from '../../../App/typer/tilkjentytelse';
 
-export const delmalTilHtml = (
-    delmalNavn: string,
-    tilkjentYtelse?: TilkjentYtelse
-): { [htmlFeltNavn: string]: string } => {
-    switch (delmalNavn) {
-        case 'varierendeInntekt':
-            return { inntektsperioderHtml: lagInntektsperioder(tilkjentYtelse) };
-        default:
-            return {};
-    }
+export const delmalTilHtml = (tilkjentYtelse?: TilkjentYtelse) => {
+    return { inntektsperioderHtml: lagInntektsperioder(tilkjentYtelse) };
 };
+
```

Comment:
Nå vil alle html-felter (er jo kun et nå da 😇 ) bli sendt med til familie-brev, uansett om de brukes i brevmalen eller ikke. Er det ønskelig?

Reply:
Kan tenkes at det er praktisk og helt fint, tenker bare på om det er en risiko i å sende rå og ustrukturert saksbehandlingsdata til en annen applikasjon uten at saksbehandler vet om det.

Reply:
Vi snakket litt om det. Blir jo litt mye ekstra data når det er mange delmaler. Alternativet er en laaaaang liste med delmalnavn i switch. Denne må holdes oppdatert og endringer i sanity vil føre til feil ved generering av brev hvis listen ikke stemmer med sanity, men jeg er usikker på hva som er best. Vi gikk for noe enkelt her, men tar gjerne en diskusjon om dette :) Kunne vi lagt inn html på "toppnivå" en gang og gjenbrukt denne i familie brev? Husker ikke helt hvordan dette er implementert.

Reply:
Ja, tror at toppnivå kunne fungert fint, og husker ikke hvorfor vi ikke gjorde det. Tror kanskje bare det var enklere å kopiere implementasjonen til flettefelt 🙈  Nå som vi bare har et html-felt tenker jeg dette er helt greit, men det blir veldig mye ubrukt data som sendes med hvis vi plutselig har 15 html-felter. Så kan vi heller vurdere å fikse senere.